paragraph/scheduling/graph_scheduler.cc

```diff
@@ -83,7 +83,7 @@ void GraphScheduler::SetLogger(std::unique_ptr<Logger> logger) {
 void GraphScheduler::InstructionStarted(
     Instruction* instruction, double current_time) {
   CHECK(initialized_);
-  CHECK_LE(current_time_, current_time);
+  // CHECK_LE(current_time_, current_time);
```

Comment:
Leaving dead code? Why are you removing this?

Reply:
updated with comment